config/locales/views/groups/en.yml

```diff
@@ -28,6 +28,10 @@ en:
     grace_day_over_limit: You have assigned one or more students who has less grace credits than the amount already used by group %{group} for this assignment.
     group_name_already_in_use: This name is already in use for this assignment.
     group_name_already_in_use_diff_assignment: The group can't be renamed, because it would result in the loss of submission data. Try a name that is not in use for another assignment.
+    groups_with_extension:
+      with_late_submission: Group With Extension and With Late Submission
```

Comment:
Please make the following wording changes:

- "Group" -> "Groups"
- "With" -> "with"
- "and With Late Submission" -> "(Late Submissions Accepted)" (to align with the individual cell contents; "With Late Submission" is actually a bit misleading, because the configuration option means that late submissions are _accepted_, but the group may or may not actually submit late).

Similar comments for the `without_late_submission` key, perhaps using "Late Submissions Not Accepted".